src/user/main.cpp

```diff
@@ -191,7 +211,12 @@ void setup() {
             #endif
 
             // We need a little delay to allow the motor time to process if it needs it
```

Comment:
Why a little delay? The motor updates are run on interrupts, meaning that this loop doesn't run while the motor is updating.